test/test-parse.cpp

```diff
@@ -48,6 +48,19 @@ TEST(test_parse, statement_starting_with_invalid_token) {
   }
 }
 
+TEST(test_parse, comma_not_allowed_between_class_methods) {
+  {
+    spy_visitor v;
+    padded_string code(u8"class f { constructor() { this._a = false; } ontext(text) { if (this._a) { process.stdout.write(text);}},}"_sv);
```

Comment:
This is an example *after* class methods. Also include *before*, and *between*. Also include a test with many consecutive commas (`class C { ,,, a(){} ,,, b(){} ,,, }`).

Reply:
Do you prefer separate test cases for each or one test that includes before, between, after, and consecutive?

Reply:
I prefer one test with only one comma (simple case), and another test with everything (for more coverage).

Reply:
Thanks, I added another test with everything.